src/mrack/utils.py

```diff
@@ -165,17 +166,25 @@ def is_windows_host(meta_host):
 
     Host is windows host if:
     * os starts with 'win' or
-    * os_type is 'windows'
+    * os_type is 'windows' or
+    * group starts with 'WIN' or
+    * group uppercase starts with 'AD'
     """
     return (
         meta_host.get("os", "").startswith("win")
         or meta_host.get("os_type", "") == "windows"
+        or meta_host.get("group", "").upper().startswith("WIN")
```

Comment:
Is the "WIN" and "AD" really needed here? I'm bit afraid that the check might trigger in correctly also on linux host where the group starts with "win" or "ad". E.g. "admin_machines" or "winners".  For that reason, the original check really does in on os name, or if somebody would have a Windows os named differently then there is the "os_type" to be explicit.

Reply:
I agree that AD could be problematic, let me remove this.